adapter/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+    kotlin("jvm")
+    id("maven-publish")
+    id("application")
+    id("com.jaredsburrows.license")
+}
+
+val debugPort = 8000
+val debugArgs = "-agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n,quiet=y"
+
+val adapterMainClassName = "org.javacs.ktda.KDAMainKt"
+
+application {
+    mainClass.set(adapterMainClassName)
+    description = "Debug Adapter for Kotlin"
+    applicationDistribution.into("bin") {
+        fileMode = 755
+    }
+}
+
+dependencies {
+	// The JSON-RPC and Debug Adapter Protocol implementations
+	implementation("org.eclipse.lsp4j:org.eclipse.lsp4j.debug:0.15.0")
+	implementation("org.jetbrains.kotlin:kotlin-stdlib")
+	implementation("org.jetbrains.kotlin:kotlin-reflect")
+	implementation("kotlin-language-server:shared")
+
+    // modules temporarily needed because of shared module import above
+    implementation("org.jetbrains.exposed:exposed-core:0.37.3")
+    implementation("org.jetbrains.exposed:exposed-dao:0.37.3")
```

Comment:
The classpath database machinery would probably be interesting for the debug adapter too, perhaps we can integrate that permanently here too?